nixos/modules/system/boot/networkd.nix

```diff
@@ -96,7 +96,7 @@ let
       "Description" "DHCP" "DHCPServer" "IPForward" "IPMasquerade" "IPv4LL" "IPv4LLRoute"
       "LLMNR" "MulticastDNS" "Domains" "Bridge" "Bond" "IPv6PrivacyExtensions"
     ])
-    (assertValueOneOf "DHCP" ["both" "none" "v4" "v6"])
```

Comment:
Nitpick about the commit message: 
It's that those were wrong. They were simply not being documented anymore. As per systemd policy you do not (immediately) remove those old options but instead drop them from the documentation ;-)

https://github.com/systemd/systemd/blob/master/src/network/networkd-network.c#L832-L840

Reply:
ah, wasn't aware they are still being parsed correctly. Will update the commit message.